setup.py

```diff
@@ -23,6 +23,35 @@ def get_version(*file_paths):
         return version_match.group(1)
     raise RuntimeError('Unable to find version string.')
 
+def load_requirements(*requirements_paths):
+    """
+    Load all requirements from the specified requirements files.
+    Returns:
+        list: Requirements file relative path strings
+    """
+    requirements = set()
+    for path in requirements_paths:
+        requirements.update(
+            line.split('#')[0].strip() for line in open(path).readlines()
```

Comment:
Is it necessary to strip comments here if there's already the check below on line 50?

```
         line.startswith('#') or
```

Reply:
I think this is to split out trailing comments after package names like:

`foo==1.0  # This is the foo package`